code/datums/ai/_ai_behavior.dm

```diff
@@ -4,8 +4,8 @@
 	var/required_distance = 1
 	///Flags for extra behavior
 	var/behavior_flags = NONE
-	///Cooldown between actions performances
-	var/action_cooldown = 0
+	///Cooldown between actions performances, defaults to the value of CLICK_CD_MELEE because that seemed like a nice standard for the speed of AI behavior
```

Comment:
if it defaults to CLICK_CD_MELEE why not just make the value the define CLICK_CD_MELEE